### PR TITLE
when-false-src

### PR DESCRIPTION
**Pull request by nicolasdelfino**
Tie the current when alt attribute closer to when by changing the name to when-false-src.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,12 +190,12 @@ Load an array of script and stylesheet resources (to be overridden).
 
 ## Advanced usage
 
-### Conditional inclusion using when
+### Conditional inclusion using when
```
Comment (gustafnk) [inline README.md]: Remove trailing space please

File: README.md [hunk 2]
```diff
@@ -190,12 +190,12 @@ Load an array of script and stylesheet resources (to be overridden).
...
 
-### Conditional inclusion using when
+### Conditional inclusion using when 
 
-When offers a way of using a predicate for inclusion. It also features an optional alt attribute that functions as the source of inclusion given that the predicate fails.
+When offers a way of using a predicate for inclusion. It also features an optional `when-alt-src` attribute that functions as the source of inclusion given that the predicate fails.
 
 ```
-<h-include src="logged-in.html" when="org.project.predicateFunction" alt="log-in.html"></h-include>
+<h-include src="logged-in.html" when-alt-src="org.project.predicateFunction" alt="log-in.html"></h-include>
```
Comment (gustafnk) [inline README.md]: Not correct...?

Comment (nicolasdelfino) [reply README.md]: yeah that was a bit off :)

File: README.md [hunk 3]
```diff
@@ -192,10 +192,10 @@ Load an array of script and stylesheet resources (to be overridden).
 
 ### Conditional inclusion using when
 
-When offers a way of using a predicate for inclusion. It also features an optional alt attribute that functions as the source of inclusion given that the predicate fails.
+When offers a way of using a predicate for inclusion. It also features an optional `when-alt-src` attribute that functions as the source of inclusion given that the predicate fails.
 
 ```
-<h-include src="logged-in.html" when="org.project.predicateFunction" alt="log-in.html"></h-include>
+<h-include src="logged-in.html" when="org.project.predicateFunction" when-alt-src="log-in.html"></h-include>
```
Comment (gustafnk) [inline README.md]: ?

**Comment by gustafnk** [comment]
PR title is when-alt-src...?

File: static/when/when-fail.html [hunk 4]
```diff
@@ -25,9 +25,9 @@
 
   <h1>h-include.js test page</h1>
 
-  <h2 id="h">When / Alt - conditional inclusion</h2>
+  <h2 id="h">When / when-alt-src - conditional inclusion</h2>
```
Comment (gustafnk) [inline static/when/when-fail.html]: Another one...

File: static/when/when-pass-use-src.html [hunk 5]
```diff
@@ -25,9 +25,9 @@
 
   <h1>h-include.js test page</h1>
 
-  <h2 id="h">When / Alt - conditional inclusion</h2>
+  <h2 id="h">When / when-alt-src - conditional inclusion</h2>
```
Comment (gustafnk) [inline static/when/when-pass-use-src.html]: Yet another one...

**Comment by gustafnk** [comment]
Yay! 🥇

**Pull request merged**